webapp/components/Comment/Comment.vue

```diff
@@ -168,6 +176,23 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    answerComment() {
+      const slug =
+        '<a class="mention" href="/profile/' +
+        this.comment.author.id +
+        '" data-mention-id="' +
+        this.comment.author.slug +
+        '" target="_blank" contenteditable="false">@d' +
+        this.comment.author.slug +
```

Comment:
Use backticks for multi-line strings. Use string interpolation.

` <- thats a backtick
${} <- this is how you can do string interpolation

Reply:
this.editor.commands.mention({ id: message.id, label: message.slug })